.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,13 @@
+# Do not allow blank issues
+blank_issues_enabled: false
```

```suggestion
blank_issues_enabled: true
```

I too like having the option for blank issues, since that's what _I_ use to file issues.